opensrp-chw-hiv/src/main/java/org/smartregister/chw/hiv/activity/BaseHivProfileActivity.kt

```diff
@@ -322,6 +324,8 @@ open class BaseHivProfileActivity : BaseProfileActivity(),
         if (!hivMemberObject.cbhsNumber.isNullOrEmpty())
             tvCbhsNumber!!.text = "CBHS:" + hivMemberObject.cbhsNumber
 
+        if (!hivMemberObject.ctcNumber.isNullOrEmpty())
```

Comment:
Lets update the visibility before setting up the ctc Number.
The same should be done for the cbhs_number. I think I missed this before

Reply:
Working on it